ghcide/src/Development/IDE/Main.hs

```diff
@@ -306,8 +308,9 @@ defaultMain Arguments{..} = do
             sessionLoader <- loadSessionWithOptions argsSessionLoadingOptions "."
             let options =
                   (argsIdeOptions argsDefaultHlsConfig sessionLoader)
-                    { optCheckParents = pure NeverCheck,
-                      optCheckProject = pure False
+                    { optCheckParents = pure NeverCheck
+                    , optCheckProject = pure False
+                    , optModifyDynFlags = pluginModifyDynflags plugins
                     }
```

Comment:
I cannot remember any valid reason for these settings, so let's just get rid of them
```suggestion
```